docs/dev/setup/localci-localvc.rst

```diff
@@ -0,0 +1,100 @@
+.. _Local CI and local VC Setup:
+
+Local CI and local VC setup
+---------------------------
+
+This section describes how to set up a programming exercise environment based on the local CI and local VC systems.
+These two systems are integrated into the Artemis server application and thus the setup is greatly simplified compared to the external options.
+This also reduces system requirements as you do not have to run any systems in addition to the Artemis server.
+For now, this setup is only recommended for development and testing purposes.
+If you are setting Artemis up for the first time, these are the steps you should follow:
+
+- Install and run Docker: https://docs.docker.com/install/
+- Start the database: :ref:`Database Setup`
+- :ref:`Configure Artemis`
+- (optional) :ref:`Configure Jira`
+- :ref:`Start Artemis`
+- :ref:`Test the Setup`
+
+.. contents:: Content of this section
+    :local:
+    :depth: 1
+
+
+.. _Configure Artemis:
+
+Configure Artemis
+^^^^^^^^^^^^^^^^^
+
+Modify ``src/main/resources/config/application-local.yml`` to include the correct URLs and directories:
```

Comment:
Just as an idea, it might be nice to have a full working configuration linked somewhere for reference. This way, any misunderstanding of any description of the config can just be eliminated by looking at the example.

Reply:
Ok, fine with me!